test/test_serialization.cpp

```diff
@@ -0,0 +1,27 @@
+// Copyright (C) 2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+
+#include "gtest/gtest.h"
+#include "ipcl/ipcl.hpp"
+
+using namespace ipcl;
```

Comment:
`using namespace ipcl` is not good, and it can not pass `cpplint`. So it woud be better to use `ipcl::xxx`

Reply:
Thanks for your suggestion.